index.html

```diff
@@ -342,7 +332,7 @@ <h1>About <br>
           </div>
         </section>
         </main>
-        <section class="contact-me">
+        <section class="contact-me" id="contact">
```

Comment:
- [ ] As the previous reviewer mentioned, please make the background of the ```contact me``` section take the full width of the webpage as in the Figma design so your design is closer to Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220604_091019](https://user-images.githubusercontent.com/25227022/171990797-bf138051-3ed1-40ce-883b-01729db8103c.png)|![Screenshot_20220604_091706](https://user-images.githubusercontent.com/25227022/171991028-be887907-6c35-4022-a365-0edad559ff01.png)|